test/src/helpers/district-set-up-utils.js

```diff
@@ -31,6 +31,28 @@ const districtSetUpUtils = {
             }
         }
         return districtSuperintendent;
+    },
+    async deleteDistrictContact(districtNumber){
+        const data = await getToken();
+        const token = data.access_token;
+        let districtSuperintendent = '';
+        let districtDetails = await districtSetUpUtils.getDistrictDetails(districtNumber);
+        let districtID = districtDetails.districtId;
+        for (const disContact of districtDetails.contacts){
```

Comment:
Please use `.filter` instead of this loop. 

See: https://github.com/bcgov/EDUC-STUDENT-ADMIN/blob/a93b9f8b48c502dd9a0230d5be1330982d3a08e2/backend/src/components/edx/exchange.js#L508

Reply:
@arcshiftsolutions I have tried using the filter function in three different styles, not one works. What am i missing? See below

       // districtSuperintendent = districtDetails.contacts.filter(contact => contact.districtContactTypeCode === 'SUPER'
       //     && contact.expiryDate === null && contact.firstName === 'Tony' && contact.lastName === 'Hawk');

        /*const filters = {districtContactTypeCode: 'SUPER', expiryDate: '', firstName: 'Tony', lastName: 'Hawk'};
        districtSuperintendent = districtDetails.contacts.filter(i =>
            Object.entries(filters).every(([k, v]) => i[k] === v));*/

         districtSuperintendent = districtDetails.contacts.filter(contact => { return contact.districtContactTypeCode === 'SUPER'
             && contact.expiryDate === '' && contact.firstName === 'Tony' && contact.lastName === 'Hawk'});

Reply:
@arcshiftsolutions this comment has been addressed and is ready for review again.